skills/lark-feed/references/lark-feed-sensitive.md

```
@@ -0,0 +1,90 @@
+      "error_code": 0,
+      "error_message": "The user is not in the chat",
+      "user_id": "ou_yyy"
+    }
+  ]
+}
+```
+
+**Pretty 模式（`--format pretty`）：**
+
+全部成功时，stdout：
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)